assets/scss/6-components/cta-block/cta-block.html

```diff
@@ -0,0 +1,42 @@
+<!-- note: strong tags in h4s only used for demo -->
+
+<!-- Social Follow CTA -->
+<div class="c-cta-block has-padding has-giant-btm-marg">
+  <div class="c-cta-block__msg t-size-b">
+    <h4 class="t-uppercase t-uppercase--extra-wide t-space-b has-text-teal has-s-btm-marg"><strong>Stay in the loop</strong></h4>
+    <p class="has-s-btm-marg t-size-l"><strong>The next time news breaks, hear about it first.</strong></p>
+    <p class="t-size-xs has-text-gray t-linkstyle t-linkstyle--underlined">Follow The Texas Tribune on <a href="#">Facebook</a> and <a href="#">Twitter</a>.</p>
+  </div>
+  <div class="c-cta-block__btns t-align-center t-size-xs">
+      <a class="c-button c-button--outline has-text-facebook has-bg-facebook has-text-hover-facebook l-align-center-children has-reset-padding" href="#" aria-label="Share on Facebook">
+        <span class="c-button__inner">Facebook</span>
+      </a>
+    <a 
+      class="c-button c-button--outline has-text-twitter has-bg-twitter has-text-hover-twitter l-align-center-children has-reset-padding" href="#" aria-label="Share on Twitter">
+        <span class="c-button__inner">Twitter</span>
+    </a>
+  </div>
+</div>
+
+<!-- Membership CTA -->
+<div class="c-cta-block has-padding">
+  <div class="c-cta-block__msg t-size-b">
+    <h4 class="t-uppercase t-uppercase--extra-wide t-space-b has-text-teal has-s-btm-marg"><strong>Join our nonprofit newsroom by becoming a member</strong></h4>
```

Comment:
Shorten example copy here to "Join our nonprofit newsroom"

Reply:
done! 👍